tests/IBFE/explicit_ex5_2d.mpirun=2.input

```diff
@@ -2,6 +2,7 @@
 Re = 200.0
 MU = 1.0/Re
 RHO = 1.0
+R = 0.5        								   // radius of the cylinder
```

Comment:
Why was this file modified? This file corresponds to `tests/IBFE/explicit_ex5.cpp` which has not been changed.

Reply:
interesting..I actually didn't manually change this. How are these tests generated? 
In the other two iim input files there is an option for prescribing the radius, and there is a call for it in the code. We can set the default value to 0.5 in example.cpp

Reply:
These tests are based on the examples but have their own source files, (here we have https://github.com/IBAMR/IBAMR/blob/master/tests/IBFE/explicit_ex5.cpp). See https://github.com/IBAMR/IBAMR/wiki/Tests-for-IBAMR for more information.

Reply:
Oh interesting. I do not remember going in there to change anything. It makes sense to revert them back to the `master` version.